docs/migration-guide.md

```diff
@@ -2,11 +2,196 @@
 
 ## Contents
 
+- [Neutrino v8 to v9](#neutrino-v8-to-v9)
 - [Neutrino v7 to v8](#neutrino-v7-to-v8)
 - [Neutrino v6 to v7](#neutrino-v6-to-v7)
 - [Neutrino v5 to v6](#neutrino-v5-to-v6)
 - [Neutrino v4 to v5](#neutrino-v4-to-v5)
 
+## Neutrino v8 to v9
+
+Neutrino v9 is our largest release ever, bringing the preset and middleware ecosystem back to the
+native tools and utilities for which they were originally created. The biggest breaking change
+is the necessity to use external tools *alongside* Neutrino now, i.e. webpack, ESLint, Jest, Karma,
+and others' native CLIs will be used in tandem with Neutrino. A list of changes is detailed below for
+migrating:
+
```

Comment:
Perhaps we should add a suggestion to refer to the differences between the output of `neutrino {build, ...} --inspect-new` vs `neutrino --inspect --mode {production, ...}` when debugging the upgrade?